StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -12,7 +12,22 @@
 import UIKit
 
 class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewControllerProtocol, PaymentSheetViewControllerProtocol {
-    var selectedPaymentOption: PaymentSheet.PaymentOption?
+    // TODO make sure `selectedPaymentOption` is correct
+    var selectedPaymentOption: PaymentSheet.PaymentOption? {
```

Comment:
I want to flesh out this relationship between `PaymentSheet.PaymentOption` <> `VerticalPaymentMethodListSelection` in a later PR. Probably in the PR where we handling when we tap Link in the wallet view.